coreos-devel/sdk-extras/sdk-extras-0.0.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2017 The CoreOS Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# This isn't meant to be installed directly. It mostly exists so we can better keep track of
+# dependencies within the SDK.
+
+EAPI=5
+
+DESCRIPTION="Meta ebuild for everything that isn't needed in the SDK, but might be useful"
+HOMEPAGE="http://coreos.com/docs/sdk/"
+SRC_URI=""
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64"
+IUSE=""
+
+DEPEND="
+	dev-util/strace
+	dev-util/cscope
+	dev-util/perf
+	dev-python/awscli
+	app-portage/repoman
+	app-portage/eix
+	dev-go/glide
+	dev-go/godep
+	app-misc/screen
+	app-misc/tmux
+	app-text/tree
+	virtual/awk
+	app-editors/nano
```

Comment:
If you're including editors, there is also a usable emacs ebuild.

Reply:
It needs `dev-libs/libunistring` (for `gnutls`). I'll add that package to portage stable I guess.

Reply:
Oh, yes, I've installed it with that `USE` flag unset.  It's probably good to get gnutls buildable, though, since we may revisit enabling it for systemd again at some point.